docs/index.md

```diff
@@ -1540,13 +1537,13 @@ mocha.setup({
 
 ### Browser-specific Option(s)
 
-The following option(s) *only* function in a browser context:
+The following option(s) _only_ function in a browser context:
 
 `noHighlighting`: If set to `true`, do not attempt to use syntax highlighting on output test code.
```

Comment:
This still exists? Thought the named negative logic functions were replaced...
```js
mocha.noHighlighting(true);
```
vs.
```js
mocha.highlighting(false);
```

Reply:
I didn't replace nor rename functions, no.  we can do that next major.

Reply:
Better stated, I thought you had added the renamed versions (retaining the negative-logic originals, but deprecating them).

Reply:
mm, I think I didn't address this because it's browser-specific; not really on the radar for CLI options.  but yeah, it needs to be addressed.  I'll make an issue.